PyDataPeek/explore_w_histograms.py

```diff
@@ -1,31 +1,92 @@
-""" 
-Script used take a csv file path, a string of numerical column names 
-    and returns a png file of histogram(s)
-
-Usage:
-  explore_w_histograms.py --file=<file_path> --columns=<column_names>
-Example:
-  python explore_w_histograms.py --file=toy_dataset.csv --columns=Age,Illness
-"""
-from docopt import docopt
 import pandas as pd
 import altair as alt
 from altair_saver import save
```

Comment:
This failed when I ran the poetry test. I believe it needs to be added to the package as a dependency?

Reply:
Oops I missed that! I deleted the docopt since I am not using it anymore.